MultiTarget/WinUI.Extra.props

```diff
@@ -33,7 +33,9 @@
     
     <RuntimeIdentifiers Condition="$([MSBuild]::GetTargetFrameworkVersion('$(TargetFramework)')) >= 8">win-x86;win-x64;win-arm64</RuntimeIdentifiers> 
     <RuntimeIdentifiers Condition="8 > $([MSBuild]::GetTargetFrameworkVersion('$(TargetFramework)'))">win10-x86;win10-x64;win10-arm64</RuntimeIdentifiers>
-    <WindowsSdkPackageVersion>10.0.22621.37-preview</WindowsSdkPackageVersion>
+
+    <WindowsSdkPackageVersion Condition="$([MSBuild]::GetTargetFrameworkVersion('$(TargetFramework)')) >= 8">10.0.22621.41</WindowsSdkPackageVersion>
+    <WindowsSdkPackageVersion Condition="8 > $([MSBuild]::GetTargetFrameworkVersion('$(TargetFramework)'))">10.0.22621.38</WindowsSdkPackageVersion>
```

Comment:
Do we have a note somewhere or docs about why this is conditional between .NET 6/8?

Reply:
See @Sergio0694's comment [here](https://github.com/CommunityToolkit/Tooling-Windows-Submodule/pull/195#discussion_r1729372053).

Reply:
Docs can be found [here](https://github.com/microsoft/CsWinRT/releases/tag/2.1.1.240807.2)